types/mui-datatables/index.d.ts

```diff
@@ -115,6 +115,11 @@ export interface MUIDataTableFilterOptions {
     logic?: (prop: string, filterValue: any[]) => boolean;
 }
 
+export interface MUIDataTableCustomFilterListOptions {
+    render?: (value: any) => string | string[];
```

Comment:
```suggestion
    render?: (value: string) => React.ReactNode;
```

From the sample the library provides it seems that the value is of type `string`: https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js

It is also often better to type the return type as ReactNode otherwise it becomes impossible to write it as: 
```ts
<Component render={value => <div>{value}</div>} />
```
This is also backed by the description:
> `render` returns a string or array of strings used as the chip label(s).

And chip labels are typed as React nodes in material-ui: https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Chip/Chip.d.ts#L12

Reply:
I don't think it is always a string parameter as I have been passing it a boolean as well in my own code.
As for the return values, I saw that in the main README.md it says `function(value) => string OR arrayOfStrings`. I'll try making the proposed changes as you have outlined and testing it.

Reply:
> I don't think it is always a string parameter as I have been passing it a boolean as well in my own code.

Ah then that's the sample putting me on the wrong track. In that case just leave it as `any`.

As for the docs.. yeah the confused me as well but they seemed to imply it was the label inside a Material UI Chip so I went by what they typed it as :\

Reply:
OK. Just tested it all with React.ReactNode and it seems to work fine. Will make the change and commit shortly.